draft-ietf-add-split-horizon-authority.xml

```diff
@@ -300,10 +303,16 @@
       possible tamperproof resolution methods are presented below.</t>
       <section anchor="validating-external">
         <name>Using a Pre-configured External Resolver</name>
-        <t>The client sends the NS query to a pre-configured resolver that is
-        external to the network, over a secure transport. Clients <bcp14>SHOULD</bcp14> apply
-        whatever acceptance rules they would otherwise apply when using this
-        resolver (e.g. checking the AD bit, validating RRSIGs).</t>
+        <t>This method applies only if the client is already configured with
+        a default resolution strategy that sends queries to a resolver outside
+        of the network over a secure transport.  That resolution strategy is
+        considered "tamperproof" because any actor who could modify the NS
+        response could already modify all of the user's other DNS responses.</t>
+        <t>To ensure that this assumption holds, clients <bcp14>MUST NOT</bcp14>
+        relax the acceptance rules they would otherwise apply when using this
+        resolver. For example, if the client would check the AD bit or
+        validate RRSIGs locally when using this resolver, it must also do so
+        when resolving NS records for this purpose.</t>
```

Comment:
We can add the following normative text:
A DNSSEC-validating client MUST apply the same validation policy to query the NS records for this purpose as it does to other queries.  A client that does not validate DNSSEC MUST apply the same policy (if any) to query the NS records as it does to other queries.

Reply:
Why is this an improvement over the text in this PR?  To me, "MUST NOT relax" seems very clear, and also more precise: in principle, I don't see a problem with clients applying a stricter validation policy to the NS probe than they do to normal resolution.  Also, there are other possible acceptance rules beyond "validate DNSSEC" or "never validate", e.g. Unbound's harden-algo-downgrade.

Reply:
I see your point. If DNSSEC is disabled by browsers for performance reasons or for validation failure due to incorrectly handling DNSSEC records by middle boxes. However, in our use case, organizations deploying split-horizon have every reason to correctly handle DNSSEC records otherwise DNSSEC validation will fail and the local resolver will not be used. DNSSEC can be conditionally enabled only for NS probe.

Reply:
Do you think the below example would be useful to add ?

Another example could be a client can do DNSSEC validation for the NS query used for this purpose even if it has disabled DNSSEC validation for other DNS queries.